src/containers/App/App.js

```diff
@@ -57,24 +71,23 @@ export class App extends Component {
         <Route path='/sign_in' component={UserLogin} />
         <Route path='/create_account' component={UserSignup} />
         <Route exact path='/genre' component={GenreContainer} />
-          {this.populateRoutes()}
-				{/* <Route path={`/${this.props.chosenGenre}`} data={this.props.movies[this.props.chosenGenre]} component={GenreContainer} /> */}
+        {this.populateRoutes()}
```

Comment:
Love how this is so dynamic now. This just pulls in all the categories and assigns them routes now, right?